.github/workflows/check.yaml

```diff
@@ -22,19 +26,30 @@ jobs:
       working-directory: ./src
 
   func:
-    uses: canonical/bootstack-actions/.github/workflows/func.yaml@v2
+    name: Functional tests
```

Comment:
I would like to know the reason that why we cannot use worflows/func.yaml@v2 or v3? 
Do we have plan to retire worflows/func.yaml@v2 ?